2025-09-29 12:48:09,431 - distributed.nanny - INFO -         Start Nanny at: 'tcp://192.168.86.107:32935'
2025-09-29 12:48:10,775 - distributed.worker - INFO -       Start worker at: tcp://192.168.86.107:43750
2025-09-29 12:48:10,775 - distributed.worker - INFO -          Listening to: tcp://192.168.86.107:43750
2025-09-29 12:48:10,775 - distributed.worker - INFO -           Worker name:             SLURMCluster-1
2025-09-29 12:48:10,775 - distributed.worker - INFO -          dashboard at:       192.168.86.107:40821
2025-09-29 12:48:10,775 - distributed.worker - INFO - Waiting to connect to: tcp://192.168.86.101:41007
2025-09-29 12:48:10,775 - distributed.worker - INFO - -------------------------------------------------
2025-09-29 12:48:10,775 - distributed.worker - INFO -               Threads:                          1
2025-09-29 12:48:10,775 - distributed.worker - INFO -                Memory:                   9.31 GiB
2025-09-29 12:48:10,775 - distributed.worker - INFO -       Local Directory: /home1/zrentala/dask-worker-space/dask-scratch-space/worker-az5ipkvf
2025-09-29 12:48:10,775 - distributed.worker - INFO - -------------------------------------------------
2025-09-29 12:48:11,104 - distributed.worker - INFO - Starting Worker plugin shuffle
2025-09-29 12:48:11,104 - distributed.worker - INFO -         Registered to: tcp://192.168.86.101:41007
2025-09-29 12:48:11,104 - distributed.worker - INFO - -------------------------------------------------
2025-09-29 12:48:11,105 - distributed.core - INFO - Starting established connection to tcp://192.168.86.101:41007
[I 2025-09-29 12:48:11,216] A new study created in memory with name: no-name-cb867289-df51-453b-90f5-d14956a33614
2025-09-29 12:48:11,236 - distributed.worker - WARNING - Compute Failed
Key:       optimize_subject-1337291c-544b-477b-8b9a-998332d7381f
Function:  execute_task
args:      ((<function optimize_subject at 0x2b3b1e3480e0>, 'Optimization_Results', 'LTP601', (<class 'dict'>, [['LTP564', (<class 'dict'>, [['recall_rate', 0.6644628099173554], ['first_recall', array([0.04878049, 0.07317073, 0.02439024, 0.07317073, 0.07317073,
       0.02439024, 0.02439024, 0.04878049, 0.        , 0.        ,
       0.02439024, 0.02439024, 0.        , 0.14634146, 0.41463415])], ['lag_crp', array([0.10714286, 0.02173913, 0.07936508, 0.15068493, 0.07058824,
       0.09183673, 0.1       , 0.06015038, 0.10958904, 0.11038961,
       0.09302326, 0.08095238, 0.11627907, 0.13807531, 0.        ,
       0.17596567, 0.12037037, 0.11578947, 0.0974359 , 0.11176471,
       0.05063291, 0.11764706, 0.06034483, 0.10576923, 0.07608696,
       0.05882353, 0.        , 0.03448276, 0.11111111])]])], ['LTP565', (<class 'dict'>, [['recall_rate', 0.7995337995337995], ['first_recall', array([0.5       , 0.        , 0.        , 0.        , 0.        ,
       0.        , 0.        , 0.        , 0.        ,
kwargs:    {}
Exception: 'NameError("name \'sub_id\' is not defined")'

[I 2025-09-29 12:48:11,238] A new study created in memory with name: no-name-f3024c9f-468e-418e-abd0-2cd72d082800
2025-09-29 12:48:11,255 - distributed.worker - WARNING - Compute Failed
Key:       optimize_subject-11a2f5be-760b-4a77-9d03-db6921f82a60
Function:  execute_task
args:      ((<function optimize_subject at 0x2b3b1e735800>, 'Optimization_Results', 'LTP583', (<class 'dict'>, [['LTP564', (<class 'dict'>, [['recall_rate', 0.6644628099173554], ['first_recall', array([0.04878049, 0.07317073, 0.02439024, 0.07317073, 0.07317073,
       0.02439024, 0.02439024, 0.04878049, 0.        , 0.        ,
       0.02439024, 0.02439024, 0.        , 0.14634146, 0.41463415])], ['lag_crp', array([0.10714286, 0.02173913, 0.07936508, 0.15068493, 0.07058824,
       0.09183673, 0.1       , 0.06015038, 0.10958904, 0.11038961,
       0.09302326, 0.08095238, 0.11627907, 0.13807531, 0.        ,
       0.17596567, 0.12037037, 0.11578947, 0.0974359 , 0.11176471,
       0.05063291, 0.11764706, 0.06034483, 0.10576923, 0.07608696,
       0.05882353, 0.        , 0.03448276, 0.11111111])]])], ['LTP565', (<class 'dict'>, [['recall_rate', 0.7995337995337995], ['first_recall', array([0.5       , 0.        , 0.        , 0.        , 0.        ,
       0.        , 0.        , 0.        , 0.        ,
kwargs:    {}
Exception: 'NameError("name \'sub_id\' is not defined")'

[I 2025-09-29 12:48:11,316] A new study created in memory with name: no-name-64fc8697-de06-498f-b6c5-7ce7e57eaba9
2025-09-29 12:48:11,332 - distributed.worker - WARNING - Compute Failed
Key:       optimize_subject-32467fee-d985-4615-b10d-bfcf6f903880
Function:  execute_task
args:      ((<function optimize_subject at 0x2b3b1e7b6ca0>, 'Optimization_Results', 'LTP571', (<class 'dict'>, [['LTP564', (<class 'dict'>, [['recall_rate', 0.6644628099173554], ['first_recall', array([0.04878049, 0.07317073, 0.02439024, 0.07317073, 0.07317073,
       0.02439024, 0.02439024, 0.04878049, 0.        , 0.        ,
       0.02439024, 0.02439024, 0.        , 0.14634146, 0.41463415])], ['lag_crp', array([0.10714286, 0.02173913, 0.07936508, 0.15068493, 0.07058824,
       0.09183673, 0.1       , 0.06015038, 0.10958904, 0.11038961,
       0.09302326, 0.08095238, 0.11627907, 0.13807531, 0.        ,
       0.17596567, 0.12037037, 0.11578947, 0.0974359 , 0.11176471,
       0.05063291, 0.11764706, 0.06034483, 0.10576923, 0.07608696,
       0.05882353, 0.        , 0.03448276, 0.11111111])]])], ['LTP565', (<class 'dict'>, [['recall_rate', 0.7995337995337995], ['first_recall', array([0.5       , 0.        , 0.        , 0.        , 0.        ,
       0.        , 0.        , 0.        , 0.        ,
kwargs:    {}
Exception: 'NameError("name \'sub_id\' is not defined")'

2025-09-29 12:48:13,636 - distributed.worker - INFO - Stopping worker at tcp://192.168.86.107:43750. Reason: scheduler-remove-worker
2025-09-29 12:48:13,638 - distributed.nanny - INFO - Closing Nanny gracefully at 'tcp://192.168.86.107:32935'. Reason: scheduler-remove-worker
2025-09-29 12:48:13,643 - distributed.nanny - INFO - Worker closed
2025-09-29 12:48:13,643 - distributed.core - INFO - Connection to tcp://192.168.86.101:41007 has been closed.
slurmstepd: error: *** JOB 3081167 ON node07 CANCELLED AT 2025-09-29T12:48:13 ***
